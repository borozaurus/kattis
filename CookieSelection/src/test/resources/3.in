8
3
1
5
2
2
6
5
#
#
#
#
#
#
#
#