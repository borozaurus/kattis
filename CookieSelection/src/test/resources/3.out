5
3
5
2
6
2
8
1